test_timeline_ancestor_detach_idempotent_success: increase timeout

Consider increasing the timeout here, unless there are signs of actual problems.

See https://neonprod.grafana.net/d/fddp4rvg7k2dcf/regression-test-failures?orgId=1&var-test_name=test_timeline_ancestor_detach_idempotent_success%5Bshards_initial_after2%5D&from=now-7d&to=now&timezone=utc&var-restrict=true&var-max_count=100&var-reference=$__all&var-ignore_reference=refs%2Fpull%2F0000%2Fmerge

```
test_runner/regress/test_timeline_detach_ancestor.py:639: in test_timeline_ancestor_detach_idempotent_success
    env.storage_controller.reconcile_until_idle()
test_runner/fixtures/neon_fixtures.py:2091: in reconcile_until_idle
    raise RuntimeError("Timeout in reconcile_until_idle")
E   RuntimeError: Timeout in reconcile_until_idle
```

--- a/test_runner/regress/test_timeline_detach_ancestor.py
+++ b/test_runner/regress/test_timeline_detach_ancestor.py
@@ -607,7 +607,7 @@ def test_timeline_ancestor_detach_idempotent_success(
 
     if shards_after > 1:
         # FIXME: should this be in the neon_env_builder.init_start?
-        env.storage_controller.reconcile_until_idle()
+        env.storage_controller.reconcile_until_idle(timeout_secs=120)
         client = env.storage_controller.pageserver_api()
     else:
         client = env.pageserver.http_client()
@@ -636,7 +636,7 @@ def test_timeline_ancestor_detach_idempotent_success(
         # Do a shard split
         # This is a reproducer for https://github.com/neondatabase/neon/issues/9667
         env.storage_controller.tenant_shard_split(env.initial_tenant, shards_after)
-        env.storage_controller.reconcile_until_idle()
+        env.storage_controller.reconcile_until_idle(timeout_secs=120)
 
     first_reparenting_response = client.detach_ancestor(env.initial_tenant, first_branch)
     assert set(first_reparenting_response) == {reparented1, reparented2}

